lib/pages/chat_list/chat_list.dart

```diff
@@ -232,6 +232,19 @@ class ChatListController extends State<ChatList>
     });
   }
 
+  void joinInviteDirectChats() {
+    Matrix.of(context).client.rooms.forEach((room) {
```

Comment:
**Attention**: Be careful when using `forEach` for `async` functions. It may work, but sometimes the results are not what you expect.

- Can we do the `joinRoom` in parallel?